python-frontend/src/test/java/org/sonar/python/tree/IPythonTreeMakerTest.java

```diff
@@ -39,21 +40,23 @@ public void line_magic() {
     var parse = parseIPython("print(b)\n" +
       "a = %alias showPath pwd && ls -a\n", treeMaker::fileInput);
     var statementList = findFirstChildWithKind(parse, Tree.Kind.STATEMENT_LIST);
-    assertThat(statementList.children()).hasSize(2);
-    assertThat(statementList.children().get(0).getKind()).isEqualTo(Tree.Kind.EXPRESSION_STMT);
-    assertThat(statementList.children().get(1).getKind()).isEqualTo(Tree.Kind.ASSIGNMENT_STMT);
-    var lineMagic = findFirstChildWithKind(statementList.children().get(1), Tree.Kind.LINE_MAGIC);
+    var statements = statementList.children();
```

Comment:
(To be honest, I kind of would have preferred to see this change on the initial PR which introduced these tests - I think it's okay to keep it as it is, but just to let you know).

Reply:
agreed